tests/runner.py

```diff
@@ -164,6 +164,12 @@ def no_windows(note=''):
   return lambda f: f
 
 
+def needs_make(note=''):
```

Comment:
Can you add an assert as the first line like the other decorators here.

Reply:
I'll fix that.